projects/observability/src/pages/explorer/explorer.component.test.ts

```diff
@@ -141,7 +141,7 @@ describe('Explorer component', () => {
         limit: 500,
         interval: new TimeDuration(15, TimeUnit.Second)
       }),
-      undefined
+      expect.objectContaining({})
```

Comment:
nit - this matches any object - we could also just put `{}` here

Reply:
weirdly enough, that doesn't work. I couldn't figure out why. Any ideas?

Reply:
What was the error? It should say what it expected/got